docs/topics/Gateway.md

```diff
@@ -677,7 +677,16 @@ A user's presence is their current state on a guild. This event is sent when a u
 |-------|------|-------------|---------|
```

Comment:
Is there a strong case for keeping this "Present" column anymore? It seems that nullable/optional notations should be clear enough for how to handle these data types.

Reply:
I agree